app/src/main/java/com/github/h3lp3rs/h3lp/locationmanager/LocationHelper.kt

```diff
@@ -52,4 +52,30 @@ class LocationHelper {
             }
         }
     }
+
+
+    /**
+     * Updates handles the user's current coordinates as wanted, or returns to the
+     * main activity in case of errors
```

Comment:
no the point is that it does not return to the main activity in case of errors, but uses the callback ;)

Reply:
woups :eyes: